docs/conf.py

```diff
@@ -64,7 +64,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build']
+# exclude_patterns = ['_build']
```

Comment:
@stevepiercy the `_build` directory is created both by `docs/Makefile` and by the tox `docs` session -- we *want* to exclude it, if present.